index.html

```diff
@@ -93,3 +93,450 @@ <h2 class="subSentenceText">I'm a Software Developer</h2>
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hesitate to
+          contact me.
+        </p>
+        <p class="AboutMeTextSection-D">
+          Hey there! I can help you build a product, feature or website. Look
+          through some of my work and experience! If you like what you see and
+          have a project you need coded, don’t hesitate to contact me.
+        </p>
+        <div>
+          <button class="ResumeButton ResumeButton-D" type="button">
+            Get My Resume
+          </button>
+        </div>
+        <div class="secondBackgroundImage"></div>
+        <div class="secondBackgroundImage-D"></div>
+        <hr class="AboutLine2" />
+        <div class="MySkills">
+          <ul class="containerTech">
+            <ul class="tech1">
+              <li class="Language1">
+                <img src="Rectangle.png" alt="Rectangle" /> Language
+              </li>
+              <ul class="TechStacks techstacks1D">
+                <li>JavaScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+              </ul>
+            </ul>
+
+            <ul class="tech2">
+              <li class="Language1">
+                <img src="Rectangle2.png" alt="Rectangle" /> Frameworks
+              </li>
+              <ul class="TechStacks">
+                <li>React.js</li>
+                <li>Ruby on Rails</li>
+                <li>RSpec</li>
+                <li>Capybara</li>
+              </ul>
+            </ul>
+
+            <ul class="tech">
+              <li class="Language1">
+                <img src="Rectangle3.png" alt="Rectangle" /> skills
+              </li>
+              <ul class="TechStacks">
+                <li>Database Management</li>
+                <li>Version Control</li>
+                <li>CLI</li>
+                <li>API Design</li>
+                <li>Web Developement</li>
+              </ul>
+            </ul>
+          </ul>
+        </div>
+      </div>
+    </section>
+
+    <!-- html form -->
+
+    <section class="Contact" id="Contact">
+      <div class="form">
+        <a href="#form"></a>
+        <div class="mainContent">
+          <h2 class="smallPara">
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h2>
+          <form
+            class="firstForm"
+            id="form"
+            method="post"
+            action="https://formspree.io/f/xgedqoye"
+          >
+
+          
+            <ul class="input">
+              <div class="subInput">
+              <li class="newInputName">
+                <input
+                  name="Name"
+                  type="Text"
+                  id="Name"
+                  placeholder="Full name"
+                  maxlength="30"
+                  required
+                />
```

Comment:
- You have done great so far. :+1: :+1: But there are some changes you still need to make before moving to the next milestone.
- [ ] As part of the requirement, you are meant to replicate the Figma design as much as possible. Looking at your UI, I think you need to adjust the height of the inputs to match the Figm design. also, the padding by the side of the form is much compared to the one in the Figma design. I'll suggest you adjust this. :smile: :wave: 

- - Your design:
![Selection_082](https://github.com/bhushan354/My-Portfolio/assets/107632780/259e94a9-bbc9-4a85-9eb1-2ba72cf330e9)

- - Figma design:
![Screenshot from 2023-07-12 15-58-36](https://github.com/bhushan354/My-Portfolio/assets/107632780/a53c8619-8ab4-4d93-bb40-1579878ea5bf)

- [ ] Following the standard practices, you are not meant to capitalize the first letter of the `type and name ` attribute in your form input. the correct attributes should be written as `type="text" name="name"`  I'll suggest you update this in your inputs. :+1: :+1: